frontend/src/component/Dashboard/RecentActivity.jsx

```diff
@@ -25,24 +27,43 @@ export const RecentActivity = props =>
       {props.recentActivityList.length > 0
         ? props.recentActivityList.map(content => {
           const contentType = props.contentTypeList.find(ct => ct.slug === content.type) || {hexcolor: '', faIcon: ''}
-          return (
+
+          const recentActivityItemIcon = (
+            <div className='recentactivity__list__item__icon' style={{color: contentType.hexcolor}}>
+              <i className={`fa fa-${contentType.faIcon}`} />
+            </div>
+          )
+
+          const recentActivityItemName = (
+            <div className='recentactivity__list__item__name'>
+              {content.label}
+              {content.type === 'file' && (
+                <Badge text={content.fileExtension} customClass='badgeBackgroundColor' />
+              )}
+            </div>
+          )
+
+          if (content.type === 'folder' && props.roleIdForLoggedUser < 4) return (
```

Comment:
What is `4` ? It seems to be a role level. This must be a constant: I guess there is other occurrence in code, so I suggest opening an issue about refactor them into constants.

Reply:
yes, the constant is in frontend, it should be moved to frontend_lib but it, indeed, requires some refactor
see https://github.com/tracim/tracim/issues/1656

Reply:
Ok thank's